Quran/AyahInfoPersistenceStorage.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  AyahInfoPersistenceStorage.swift
+//  Quran
+//
+//  Created by Ahmed El-Helw on 5/12/16.
+//  Copyright © 2016 Quran.com. All rights reserved.
+//
+
+import Foundation
+import SQLite
+
+struct AyahInfoPersistenceStorage {
+
+    private let db: Connection
+    private let glyphsTable = Table("glyphs")
+
+    private let idColumn = Expression<Int>("glyph_id")
+    private let pageColumn = Expression<Int>("page_number")
+    private let suraColumn = Expression<Int>("sura_number")
+    private let ayahColumn = Expression<Int>("ayah_number")
+
+    private let imageSize = "1920"
+
+    init() throws {
+        let file = String(format: "images_\(imageSize)/databases/ayahinfo_\(imageSize)")
+        guard let path = NSBundle.mainBundle().pathForResource(file, ofType: "db") else { fatalError() }
+        db = try Connection(path, readonly: true)
```

Comment:
I would like to wrap that exception into a PersistenceError. Also, what about lazily initialize the connection.